ui/ui-components/pages/resque/overview.tsx

```diff
@@ -214,11 +214,25 @@ export default function ResqueOverview(props) {
                     return (
                       <tr key={name}>
                         <td>
-                          <span
-                            className={worker.delta > 0 ? "text-success" : ""}
+                          <Link
+                            href={
+                              "/resque/workers#" +
+                              `${worker.host}:${worker.id}`.replace(
+                                /[\W_-]/g,
+                                "-"
+                              )
```

Comment:
The worker names have `:` and `.` in them - we need to clean them up to be valid HTML anchors

Reply:
👍  I don't think we need the "-" to be replaced if the replacement is "-" though.